tagstudio/src/qt/helpers/file_deleter.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import os.path
```

Comment:
please try to use `pathlib` instead of `os.path`, see [CONTRIBUTING.md#implementations](https://github.com/TagStudioDev/TagStudio/blob/main/CONTRIBUTING.md#implementations)